Ical.Net.Tests/Calendars/Recurrence/RecurrenceTestCases.txt

```diff
@@ -181,3 +181,94 @@ INSTANCES:20250101,20251231,20260101
 DTSTART:20250601
 RRULE:FREQ=YEARLY;BYMONTH=6;BYMONTHDAY=2,-2;UNTIL=20250630
 INSTANCES:20250602,20250629
+
+############################## START ERRATA 1913 TESTS ##############################
+# RFC errata 1913 (https://www.rfc-editor.org/errata_search.php?rfc=1913&amp;eid=1913):
+# 'The numeric value in a BYDAY rule part with the FREQ rule part set to YEARLY corresponds  
+# to an offset within the month when the BYMONTH rule part is present, and corresponds to an 
+# offset within the year when the BYWEEKNO or BYMONTH rule parts are NOT present.'
+# 
+# These 12 tests verify the behavior with the interpretation that
+# * when only BYMONTH is present, the numeric BYDAY offset applies within the month,
+# * when BYWEEKNO is present, the numeric BYDAY offset applies within the week.
```

Comment:
Applying the numeric offset within the week is a reasonable interpretation following the logic of the case that `BYMONTH` is present. However, as the RFC is quite clear that BYWEEKNO + numeric BYDAY offset is not allowed, I'd rather suggest to throw in this case. Or are there cases in the wild that force us to deal with this case?

Moreover, as 1 would be the only reasonable integer (even -1 wouldn't be allowed, as every week always has 7 days, so there is no need for -1 as it isn't needed and therefore not allowed for other components like hours, which don't vary in different intervals) there isn't much point in using it, so any use of it would most likely be a usage error anyways.

Reply:
Will add "RFC says that BYWEEKNO + numeric BYDAY is invalid. We're taking the lenient approach here."